src/datasets/bigquery/clients_last_seen/intro.md

```diff
@@ -0,0 +1,41 @@
+The `clients_last_seen` dataset is useful for determining exact user counts
+such as MAU, WAU, and DAU.
+
+The `clients_last_seen` dataset is similar to the
+[`client_count_daily` dataset](/datasets/batch_view/client_count/reference.md)
+except it is used to efficiently generate exact counts, and it includes the most
+recent values in a 28 day window for all columns from the
```

Comment:
When would one prefer `client_count_daily` to `clients_last_seen`?

Reply:
`clients_last_seen` isn't available outside bigquery. Otherwise I can think of no reason not to deprecate `client_count_daily` in favor of `clients_last_seen`